inc/admin-ui.php

```diff
@@ -307,3 +308,126 @@ function remove_author_support() {
 	}
 }
 add_action( 'admin_init', __NAMESPACE__ . '\remove_author_support' );
+
+/**
+ * Add filters to control columns on the post edit screen.
+ */
+function maybe_modify_post_edit_columns() {
+	if ( apply_filters( 'byline_manager_modify_post_edit_columns', true ) ) {
+		$supported_post_types = Utils::get_supported_post_types();
+		foreach ( $supported_post_types as $post_type ) {
+			add_filter( "manage_{$post_type}_posts_columns", __NAMESPACE__ . '\augment_author_column' );
+			add_action( "manage_{$post_type}_posts_custom_column", __NAMESPACE__ . '\render_byline_column', 10, 2 );
+		}
```

Comment:
is there a use case where we would want to add this column on one supported post type but not another? Just curious on if we should add another filter either around `Utils::get_supported_post_types();` or inside the `foreach`
```
$supported_post_types = apply_filters( 'byline_manager_edit_columns_post_types', Utils::get_supported_post_types() );
```
or
```
		foreach ( $supported_post_types as $post_type ) {
			if ( apply_filters( 'byline_manager_modify_posttype_edit_columns', true, $post_type ) ) {
						add_filter( "manage_{$post_type}_posts_columns", __NAMESPACE__ . '\augment_author_column' );
						add_action( "manage_{$post_type}_posts_custom_column", __NAMESPACE__ . '\render_byline_column', 10, 2 );
			}
		}
```